Client/game_sa/CGameSA.cpp

```diff
@@ -628,17 +565,12 @@ bool CGameSA::SetCheatEnabled(const char* szCheatName, bool bEnable)
 
 void CGameSA::ResetCheats()
 {
-    SetRandomFoliageEnabled(true);
-    SetMoonEasterEggEnabled(false);
-    SetExtraAirResistanceEnabled(true);
-    SetUnderWorldWarpEnabled(true);
-    SetCoronaZTestEnabled(true);
-    CVehicleSA::SetVehiclesSunGlareEnabled(false);
-    SetWaterCreaturesEnabled(true);
```

Comment:
Where do you reset the world's special properties?

Reply:
> Where do you reset the world's special properties?

In [CPacketHandler::Packet_MapInfo](https://github.com/multitheftauto/mtasa-blue/pull/3161/files#diff-ca2b02caf256208d2211a21b062c3202b7334801dffa4106bbf052ab3c85a12fR2355-R2371). Didn't want to add default states in one more place and decided to go directly with sync structure (default state overrides in [SWorldSpecialPropertiesStateSync](https://github.com/multitheftauto/mtasa-blue/pull/3161/files#diff-e6465dfacbeedc94f3e8b4348f93f45f9a5b43511786b2e99834afefcf2cf04aR2073-R2079)). Tested it with updated / not updated server.